builtin/settingtypes.txt

```diff
@@ -265,8 +265,8 @@ undersampling (Undersampling) int 1 1 8
 
 [**Graphics Effects]
 
-#    Makes all liquids opaque
-opaque_water (Opaque liquids) bool false
+#    Makes water translucent
```

Comment:
"Makes water translucent" isn't true. The setting doesn't make water transparent, but merely allows it to be transparent. If a game has opaque water, it still won't be transparent even if the setting is enabled.

Reply:
Ah yes... I will correct that

Reply:
(btw, for the future: you can resolve conversations once you have addressed the review points)